### PR TITLE
Add course page

### PR DESCRIPTION
**Pull request by JohnHilding**

### CODE REVIEW

**Comment by RobertBrunhage** [comment]
Noice!

What I've seen so far that needs fixes.

1. The route and link should be **course** and not **courses**
2. The about me section seems more indented the the other sections
3. The "WHAT YOU GET" box is not rounded

**Comment by RobertBrunhage** [comment]
- The landing page seems a bit affected with the font size change
image

- Change the text on buttons from "ENROLL NOW" to what we currently have on Figma
- The last button on mobile should be below the "When you sign up you are also eligible to win this course" text

File: pages/course.tsx [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+const courses = () => {
+  return (
+    <PlausibleProvider domain="robertbrunhage.com">
+      <Layout>
+        <Head>
+          <title>Robert Brunhage - Flutter, Dart, Firebase | Courses</title>
+          <link rel="icon" href="/favicon.ico" />
+          <meta name="robots" content="index" />
+          <meta name="description" content="Flutter Courses | Robert Brunhage" />
+          <meta property="og:url" content="https://robertbrunhage.com/courses" />
```
Comment (RobertBrunhage) [inline pages/course.tsx:19]:
```suggestion
          <meta property="og:url" content="https://robertbrunhage.com/course" />
```

File: pages/course.tsx [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+          />
+          <meta property="twitter:card" content="summary" />
+          <meta property="twitter:site" content="@robertbrunhage" />
+          <meta property="twitter:title" content="RobertBrunhage.com" />
+          <meta property="twitter:description" content="Flutter Courses" />
+          <meta
+            property="twitter:image"
+            content="https://robertbrunhage.com/assets/images/course_twitter.png"
+          />
+          <link rel="prefetch" href="https://robertbrunhage.com/videos" />
```
Comment (RobertBrunhage) [inline pages/course.tsx:35]:
```suggestion
          <link rel="prefetch" href="https://robertbrunhage.com/course" />
```

File: pages/course.tsx [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+          <link rel="canonical" href="https://robertbrunhage.com" />
```
Comment (RobertBrunhage) [inline pages/course.tsx:36]:
```suggestion
          <link rel="canonical" href="https://robertbrunhage.com/course" />
```

File: pages/course.tsx [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+              </p>
+              <p>
+                <strong>
+                  What I teach in this course is knowledge that I’ve accumulated for more than 3
+                  years of Flutter development.
+                </strong>
+              </p>
+              <p>
+                Whether you feel like an beginner (as long as you know the basics), intermediate or
+                expert this course fasttrack your Flutter journey.
```
Comment (RobertBrunhage) [inline pages/course.tsx:74]:
```suggestion
                expert this course will fast track your Flutter journey.
```

File: pages/course.tsx [hunk 5]
```diff
@@ -0,0 +1,132 @@
...
+              <li>learn advanced concepts</li>
+              <li>project you can build upon</li>
+            </ul>
+          </section>
+          <section>
+            <div className={`max_width ${styles.about}`}>
+              <div>
+                <h2>Hey! 👋</h2>
+                <p>
+                  I’m Robert Brunhage. You may know me by my <strong>YouTube channel</strong> or
```
Comment (RobertBrunhage) [inline pages/course.tsx:101]: Make the YouTube channel text a link to the channel

**Review by RobertBrunhage — CHANGES_REQUESTED** [review]
Some small things I noticed.

File: pages/course.tsx (hunk 2 above)
Comment (JohnHilding) [reply pages/course.tsx:35]: no

**Review by RobertBrunhage — APPROVED** [review]
Nice, tyty

File: components/emailForm/forms/emailList.tsx [hunk 6]
```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import Newsletter from "../../newsletter/newsletter";
+
+const emailList = ({ color }) => {
+  return (
+    <div>
+      <Newsletter
+        color={color}
+        title="Get my free Flutter Tips PDF"
+        description="Level up your skills by joining 600 other Flutter developers now! 😃"
```
Comment (RobertBrunhage) [inline components/emailForm/forms/emailList.tsx:10]:
```suggestion
        description="Level up your skills by joining 1000 other Flutter developers now! 😃"
```

File: pages/course.tsx [hunk 7]
```diff
@@ -0,0 +1,141 @@
...
+          <meta property="og:title" content="RobertBrunhage.com" />
+          <meta property="og:description" content="Flutter Courses" />
+          <meta
+            property="og:image"
+            content="https://robertbrunhage.com/assets/images/course_twitter.png"
+          />
+          <meta property="twitter:card" content="summary" />
+          <meta property="twitter:site" content="@robertbrunhage" />
+          <meta property="twitter:title" content="RobertBrunhage.com" />
+          <meta property="twitter:description" content="Flutter Courses" />
```
Comment (RobertBrunhage) [inline pages/course.tsx:30]:
```suggestion
          <meta property="twitter:description" content="Build a complete production ready Flutter application!" />
```

File: pages/course.tsx [hunk 8]
```diff
@@ -0,0 +1,141 @@
...
+          <section className={styles.header}>
+            <h1>The Ultimate Flutter Course</h1>
+            <h3>Build a complete production ready Flutter application</h3>
+            <div className={styles.form}>
+              <FormInput
+                giveaway={""}
+                cta={"get updates"}
+                color={"var(--primary-bg-color)"}
+                action={
+                  "https://gmail.us2.list-manage.com/subscribe/post?u=ff73d806dd2f49da87ede8337&amp;id=ed4e712aca&amp;SIGNUP=FlutterMovieCourse"
```
Comment (RobertBrunhage) [inline pages/course.tsx:48]: Make this action url a const, and replace all the magic strings with the const

File: pages/course.tsx [hunk 9]
```diff
@@ -0,0 +1,141 @@
...
+                Whether you feel like an beginner (as long as you know the basics), intermediate or
+                expert this course will fast track your Flutter journey.
+              </p>
+              <div className={styles.form}>
+                <FormInput
+                  giveaway={""}
+                  cta={"get updates"}
+                  color={"var(--secondary-bg-color)"}
+                  action={
+                    "https://gmail.us2.list-manage.com/subscribe/post?u=ff73d806dd2f49da87ede8337&amp;id=ed4e712aca&amp;SIGNUP=FlutterMovieCourse"
```
Comment (RobertBrunhage) [inline pages/course.tsx:82]: Same here

**Review by RobertBrunhage — CHANGES_REQUESTED** [review]
Some more things :)

**Review by RobertBrunhage — APPROVED** [review]

**Pull request merged**